### PR TITLE
Ex34/create main page layout

### PR DESCRIPTION
**Pull request by fbozhkov**

### CODE REVIEW

File: client/src/components/main-layout/mainPage.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                            <ProductCard></ProductCard>
+                        </Grid>
+
+                        <Grid item xs={12} sm={6} md={4}>
+                            <ProductCard></ProductCard>
+                        </Grid>
+
+                        <Grid item xs={12} sm={6} md={4}>
+                            <ProductCard></ProductCard>
+                        </Grid>
```
Comment (JordanRad) [inline client/src/components/main-layout/mainPage.js]: Please, use an array instead

File: client/src/components/main-layout/mainPage.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                                    <ProductCard></ProductCard>
+                                </Grid>
+
+                                <Grid item xs={12} sm={6} md={4}>
+                                    <ProductCard></ProductCard>
+                                </Grid>
+
+                                <Grid item xs={12} sm={6} md={4}>
+                                    <ProductCard></ProductCard>
+                                </Grid>
```
Comment (JordanRad) [inline client/src/components/main-layout/mainPage.js]: Same here

**Review by JordanRad — CHANGES_REQUESTED** [review]

File: client/src/components/main-layout/mainPage.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
+@import'../../shared/styles/colors';
+
+.main {
+    max-width: 1536px; // equal to background image width
```
Comment (petya0111) [inline client/src/components/main-layout/mainPage.scss:4]: 1. fixed width can make problems in responsive views
2. I recently had an image focal point (for the image responsiveness) task on my full-time job and now I think something like that will be good https://www.npmjs.com/package/image-focus 
3. or if needed take more large images from designer

**Review by petya0111 — CHANGES_REQUESTED** [review]

**Review by IvayloSlavchev — CHANGES_REQUESTED** [review]
Remove the code in scss files that you're not going to use now.

**Comment by fbozhkov** [comment]
fixed

**Review by JordanRad — APPROVED** [review]
LGTM

**Review by petya0111 — APPROVED** [review]
Fully functional and responsive, good to go!  ✔️

**Review by IvayloSlavchev — APPROVED** [review]
LGTM!

**Pull request merged**